### PR TITLE
Tweak: Setting up the Game Server

### PR DESCRIPTION
**Pull request by Kalaiselvi84**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, press enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind breaking
> /kind bug
> /kind cleanup
> /kind documentation
> /kind feature
> /kind hotfix
> /kind release

**What this PR does / Why we need it**:

**Which issue(s) this PR fixes**:

Closes #

**Special notes for your reviewer**:

### CODE REVIEW

File: site/content/en/docs/Examples/simple-genai-gameserver.md [hunk 1]
```diff
@@ -40,28 +40,12 @@ on that cluster, otherwise install Agones into your GenAI inference server clust
...
+
+To make two clients "chat" to each other, change the `GEN_AI_ENDPOINT` and `SIM_ENDPOINT` values in the [gameserver_autochat.yaml](https://github.com/googleforgames/agones/blob/main/examples/simple-genai-server/gameserver_autochat.yaml) file to your inference server's address. You could also create a basic http server that receives requests in the structure noted in the above section, and returns a predetermined set of responses for the chat. When sending requests, `GEN_AI_CONTEXT` is sent
+to the `GEN_AI_ENDPOINT` and `SIM_CONTEXT` is sent to the `SIM_ENDPOINT` as part of the GenAIRequest structure. The default values for `GEN_AI_CONTEXT` and `SIM_CONTEXT` are empty strings. The conversation starts with a `PROMPT` that is sent to the GenAI endpoint, which is also an empty string by default. `NUM_CHATS` determines how many times messages are sent back and forth, with a default value of 1.
```
Comment (igooch) [inline site/content/en/docs/Examples/simple-genai-gameserver.md:46]: We should make a note here that there is a rolling conversation, so after a conversation of length `NUM_CHATS` a new conversations starts. This continues until the game server is deleted with `kubectl delete`.

Comment (igooch) [reply site/content/en/docs/Examples/simple-genai-gameserver.md:46]: Note that this is slightly different than the behavior with NPC Chat. In NPC Chat a new conversation is started either when it reaches the length `NUM_CHATS` or when is reaches the `STOP_PHRASE`, whichever comes sooner. As with the auto chat, this is a rolling set of conversations that continues until the game server is deleted.

**Review by igooch — COMMENTED** [review]
Quick comment on `NUM_CHATS`, otherwise looks good!

File: site/content/en/docs/Examples/simple-genai-gameserver.md (hunk 1 above)
Comment (Kalaiselvi84) [reply site/content/en/docs/Examples/simple-genai-gameserver.md]: I've updated it as per your suggestion.

**Comment by Kalaiselvi84** [comment]
🤔I think Mark will likely submit another PR to update this document.🤔

**Review by igooch — APPROVED** [review]
LGTM

**Pull request merged**